README.md

```diff
@@ -23,6 +23,8 @@ The Example project uses [Carthage](https://github.com/Carthage/Carthage) to int
 
 Then, in your checkout of the `NYTPhotoViewer` repo, run `carthage checkout --use-submodules`.
 
+Then run `scripts/bootstrap`.
```

Comment:
To make it friendlier to beginners:

```suggestion
Then run `./scripts/bootstrap`.
```

Also, I _think_ the bootstrap script should obviate the need for the Carthage checkout from the previous line? Not sure, though.

Reply:
Good question. I tried it with a fresh `clone` and the answer is "bootstrap is sufficent".